util/topgen/merge.py

```diff
@@ -863,12 +883,16 @@ def create_alert_lpgs(top, name_to_block: Dict[str, IpBlock]):
             # indication signals, we can just use the clock / reset signals
             # of the first block that belongs to a new unique LPG.
             clock = module['clock_connections'][block_clock.clock]
-            top['alert_lpgs'].append({
+            lpg = {
                 'name': lpg_name,
-                'clock_group': clock_group,
```

Comment:
Why was clock_group removed from this dict? That changes the logic for non-external clocks.